test/container.test.ts

```diff
@@ -809,7 +813,34 @@ test('custom security context', () => {
   expect(container.securityContext.group).toEqual(2000);
   expect(container.securityContext.capabilities?.add).toEqual(['AUDIT_CONTROL']);
   expect(container.securityContext.capabilities?.drop).toEqual(['BPF']);
+  expect(container.securityContext.seccompProfile?.type).toEqual('RuntimeDefault');
+
+});
 
+test('seccompProfile localhostProfile can not be used if type is not Localhost', () => {
+
+  const container = new Container({
+    image: 'image',
+    securityContext: {
+      seccompProfile: {
+        type: SeccompProfileType.LOCALHOST,
+        localhostProfile: 'localhostProfile',
+      },
+    },
+  });
+
+  const spec = container._toKube();
+  expect(spec.securityContext?.seccompProfile?.localhostProfile).toEqual('localhostProfile');
+
+  expect(() => new Container({
+    image: 'image',
+    securityContext: {
+      seccompProfile: {
+        type: SeccompProfileType.UNCONFINED,
+        localhostProfile: 'localhostProfile',
+      },
+    },
+  })).toThrowError('localhostProfile must only be set if type is "Localhost"');
```

Comment:
Please put this in a separate test.

Reply:
my mistake, sorry about that. I sent a commit to separate tests